### ISSUE TITLE
[Help wanted] Error while Creating the Training and Validation Datasets

### ISSUE THREAD

**Issue by MaxVD76** [akTwelve/cocosynth#17]
Hi,

I'am currently trying to detect weeds in sugar beet field. I used akTwelve cocosynth codes to generate my dataset (10,000 training images and 1,000 validation images). Thanks a lot, those codes are amazing.
Now I'm trying to train the mask-rcnn model adapted from Matterport repo and an issue occurred when I give the path of the coco_instances.json files ("Create the Training and Validation Datasets", in cell 11).  

![image](https://user-images.githubusercontent.com/65913566/83130206-404a0c80-a0de-11ea-8673-dd92c6a65c31.png)

As you can see it seems to be an error in "load_data" while reading "image_id" in the json file. 
In those file I noticed that at the end of the file lots of commas are "messy" (for big datasets like the training one). I generated small dataset (with "clean" jsons files) and the issue in the code was the same, so it seems not to be the issue.

![image](https://user-images.githubusercontent.com/65913566/83130983-72a83980-a0df-11ea-81f2-3b39bf5d49d8.png)

I precise that the training code work perfectly with the jsons files from the "box_dataset_synthetic_complete" that akTwelve made.

I found no help anywhere on the web and my teacher do not understand the error to.
Does anyone have an idea to fix this trouble and help me?

Thanks a lot for helping me!

Best
Maxime

**Comment by akTwelve** [comment]
I'm sorry, I'm not sure what the issue would be. The screenshot of your json file is very strange, do you know why the numbers are overlapping like that?

**Comment by MaxVD76** [comment]
Thank you for your answer. No I don't know... I think that is just an error on displaying.
The good news is that I found a way to get around this error yesterday.
The issue is caused by a lack in the json files (some pictures are called but nothing is assigned to these). I deleted the calls and it runs! (I don't know why there are some lacks, I tried to generate the json on several computers and the results were the same...).
Anyway, it is working fine (I am currently training the mask-rcnn on colab) and thank yo again for your codes, it works quite fine!!
(If you're interested in our results don't hesitate to e-mail me).
